stable/rabbitmq-ha/values.yaml

```diff
@@ -133,6 +133,13 @@ rabbitmqNodePort: 5672
 ## Manager port
 rabbitmqManagerPort: 15672
 
+## Customize nodePort number when service type is NodePort
+### Ref: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
+###
+rabbitmqEpmdNodePort: null
```

Comment:
Since these ports only apply to the service, let's move/rename them to `service.epmdNodePort`, `service.amqpNodePort` and `service.managerNodePort`

Reply:
Let's drop the rabbitmq prefix and use ampq instead of node: `service.epmdNodePort`, `service.amqpNodePort` and `service.managerNodePort`

Reply:
Sorry, I missed that one earlier. It's done now.